File: S1/C#Project/C#作业/2月26日周末作业/MyTime.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _2月26日周末作业
{
	class MyTime
	{
		public double Times;
        public string getInfo()
		{
			string time = "";
			if (Times >= 5 && Times <= 10)
			{
				time = "早晨";
			}
			else if (Times >= 11 && Times <= 13)
			{
				time = "中午";
			}
			else if (Times >= 13 && Times <= 19)
			{
				time = "下午";
			}
			else if (Times >= 18 && Times <= 24)
			{
				time = "晚上";
			}
			else
			{
				time = "清晨";
			}
			return time;
		}
	}

    class TestMyTime
    {
		static void Main(string[] args)
        {
			MyTime time = new MyTime();
			Console.WriteLine("请输入您的时间：");
			time.Times = int.Parse(Console.ReadLine());
			string Tim = time.getInfo();
			Console.WriteLine(Tim);
			Console.ReadLine();
		}
	}
}

File: S1/C#Project/Java/CH12/11月19日作业/cn/jbit/MyTime.java
package cn.jbit;

public class MyTime {
	double times;
	public String getInfo(){
		String time = "";
		if(times >= 5 && times <= 10){
			time = "?糿";
		}else if(times >= 11 && times <= 13){
			time = "????";
		}else if(times >= 13 && times <= 19){
			time = "????";
		}else if(times >= 18 && times <= 24){
			time = "????";
		}else{
			time = "?峿";
		}
		return time;
	}
}
